hook-samples/module-save-time-logger/README.md

```diff
@@ -0,0 +1,3 @@
+# Hook sample for Polarion Interceptor extension
+
+This hook writes to log module save time.
```

Comment:
_This hook writes module save time to log._
or
_This hook logs module save time._

Reply:
Fixed